evap/results/views.py

```diff
@@ -229,9 +213,11 @@ def evaluation_detail(request, semester_id, evaluation_id):
         is_contributor=evaluation.is_user_contributor(view_as_user),
         is_responsible_or_contributor_or_delegate=evaluation.is_user_responsible_or_contributor_or_delegate(view_as_user),
         can_download_grades=view_as_user.can_download_grades,
+        can_export_text_answers=(not view_as_user.is_student and view in ("export", "full")),
```

Comment:
instead of using `is_student` (which is true for any user who participated in any course - we can't assume students and contributors are disjoint) this should probably be something like `is_user_responsible_or_contributor_or_delegate` for this evaluation (see method in `Evaluation`).